generators/entity/index.js

```diff
@@ -925,6 +932,33 @@ module.exports = EntityGenerator.extend({
             },
             {
                 when: function (response) {
+                    return response.relationshipAdd === true &&
+                        response.relationshipType === 'many-to-one';
```

Comment:
required could be set for one-to-one as well rite? and technically I can even have required for other types of relationships as well